lib/tasks/graphql.rake

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Note: Make dumps of schema for different environments - public/private!
+
+namespace :graphql do
+  desc "Write a dump of the entire GraphQL Schema"
+  task(dump_schema: :environment) do
+    # Get a string containing the definition in GraphQL IDL:
+    schema_defn = MushroomObserverSchema.to_definition
+    # Choose a place to write the schema dump:
+    schema_path = "app/graphql/schema.graphql"
```

Comment:
This may be picky, but can I assume that schema.graphql is more data than code?  I feel like data does not belong in the app tree.  Maybe this belongs in public or ... beats me! somewhere else ... even the rails root would be preferable (I already have a bunch of crud in my rails root, for example, but I keep everything else but public clean).

Reply:
Moving it to /public/graphql :)